src/test/kotlin/uk/gov/dluhc/notificationsapi/config/IntegrationTest.kt

```diff
@@ -152,7 +156,7 @@ internal abstract class IntegrationTest {
 
     @BeforeEach
     fun clearMessagesFromStubs() {
-        updateStatisticsMessageListenerStub.clear()
+        updateVoterCardStatisticsMessageListenerStub.clear()
```

Comment:
Do we want to clear postal stats messages here as well?

Reply:
Good shout!